railties/lib/rails/generators/erb/scaffold/templates/show.html.erb.tt

```diff
@@ -1,19 +1,10 @@
 <p id="notice"><%%= notice %></p>
 
-<% attributes.reject(&:password_digest?).each do |attribute| -%>
-<p>
-  <strong><%= attribute.human_name %>:</strong>
-<% if attribute.attachment? -%>
-  <%%= link_to @<%= singular_table_name %>.<%= attribute.column_name %>.filename, @<%= singular_table_name %>.<%= attribute.column_name %> if @<%= singular_table_name %>.<%= attribute.column_name %>.attached? %>
-<% elsif attribute.attachments? -%>
-  <%% @<%= singular_table_name %>.<%= attribute.column_name %>.each do |<%= attribute.singular_name %>| %>
-    <div><%%= link_to <%= attribute.singular_name %>.filename, <%= attribute.singular_name %> %></div>
-  <%% end %>
-<% else -%>
-  <%%= @<%= singular_table_name %>.<%= attribute.column_name %> %>
-<% end -%>
-</p>
+<%%= render @<%= singular_table_name %> %>
 
-<% end -%>
-<%%= link_to 'Edit', edit_<%= singular_table_name %>_path(@<%= singular_table_name %>) %> |
-<%%= link_to 'Back', <%= index_helper %>_path %>
+<div>
+  <%%= link_to "Edit this <%= human_name.downcase %>", edit_<%= singular_table_name %>_path(@<%= singular_table_name %>) %> |
+  <%%= link_to "Back to <%= human_name.pluralize.downcase %>", <%= index_helper %>_path %>
+
+  <%%= button_to "Destroy this <%= human_name.downcase %>", <%= singular_table_name %>_path(@<%= singular_table_name %>), method: :delete %>
```

Comment:
Maybe `link_to` would be better? 

```suggestion
  <%%=link_to "Destroy this <%= human_name.downcase %>", <%= singular_table_name %>_path(@<%= singular_table_name %>), method: :delete, data: { confirm: 'Are you sure?' } %>
```

Reply:
We are looking to phase out links that perform DELETE. It's not a good, safe pattern.